### PR TITLE
Fixed rendering time

### LINKED ISSUE #6: Take some minutes to render elements.
Need to reduce the time of rendering elements. If you could suggest any suggestions discuss here then we will assign you.

### PR DESCRIPTION
**Pull request by ad-dan**
Fixes #6 

One of the biggest reasons for load times was accessing the language color code API. Fixed it by caching in local storage.

The second biggest reason was accessing each repo's details using individual requests. All those requests really add up. Fixed by querying the user's repositories API endpoint and fetching all required details in one go.

### CODE REVIEW

File: src/components/GitHubCards.js [hunk 1]
```diff
@@ -1,64 +1,40 @@
...
-      baseURL(`covid-19-tracker`),
-      baseURL(`material-ui-medium-blog`),
-      baseURL(`github-readme-medium-card`),
-      baseURL(`react-youtube-search-clone`),
-      baseURL(`tamil-song-corpus`),
-      baseURL(`Nozama_Warriors`),
-      baseURL(`Nursery_Management`),
-      baseURL(`React-Medium-Blog`),
-    ];
+    let repos = await Axios.get(`https://api.github.com/users/${USER_NAME}/repos`);
```
Comment (ssat4805) [inline src/components/GitHubCards.js:17]: It is fetching all repositories. The aim of this project is to fetch specific repos. If user have 50 or 60 repositories it is not worth. Could you please restructure it

**Review by ssat4805 — CHANGES_REQUESTED** [review]

File: src/components/GitHubCards.js (hunk 1 above)
Comment (ssat4805) [reply src/components/GitHubCards.js:17]: user likes to only show his popular repositories on their website. Thank you for your PR. Nice work

**Comment by ad-dan** [comment]
Shows top 8 repos now

**Comment by ssat4805** [comment]
Thank you @ad-dan . Nice work ❤❤❤

**Comment by ssat4805** [comment]
@ad-dan  I made Some fixes. watch it. Don't forget to give a star

**Pull request merged**